lib/hello_web/router.ex

```diff
@@ -0,0 +1,101 @@
+defmodule HelloWeb.Router do
+  use HelloWeb, :router
+
+  pipeline :browser do
+    #overridings rendering format
+    plug :accepts, ["html", "text"]
+    plug :fetch_session
+    plug :fetch_live_flash
+    plug :put_root_layout, {HelloWeb.LayoutView, :root}
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+    plug HelloWeb.Plugs.Locale, "en"
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  #scope routes-> group of routes under a common path prefix
+  scope "/admin", HelloWeb.Admin, as: :admin do
+    pipe_through :browser
+
+    resources "/reviews", ReviewController
```

Comment:
@keliumJU could you explain what a resource is?

Reply:
A resource is an HTTP macro that allows me to create a basic structure of routes following the RESTful architecture, that is, routes for creating, updating, displaying and deleting data (CRUD)